spring-integration-core/src/main/java/org/springframework/integration/handler/AbstractReplyProducingMessageHandler.java

```diff
@@ -159,6 +171,18 @@ public String toString() {
 			return AbstractReplyProducingMessageHandler.this.toString();
 		}
 
+		/**
+		 * Utility method, intended for use in message handler advice classes to get
+		 * information about the advised object. For example:
+		 * <p>
+		 * {@code ((AbstractReplyProducingMessageHandler.RequestHandler)
```

Comment:
I was originally going to do that but since it's public, it would be a breaking change - although I suppose the chance that someone used the interface somehow would be extremely small.